front-end/src/app/navegacao/adm/adm-menu/adm.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-adm',
+  templateUrl: './adm.component.html'
+
+})
+export class AdmComponent implements OnInit {
```

Comment:
novamente, nome bosta

Reply:
Ja mudei no outro branch q to fazendo os graficos